src/App.jsx

```diff
@@ -1,299 +1,38 @@
-import React, { Component } from 'react';
-import { connect, Provider } from 'react-redux';
-import PropTypes from 'prop-types';
-import { IntlProvider, injectIntl, intlShape } from 'react-intl';
+import React from 'react';
 import { ConnectedRouter } from 'connected-react-router';
-import { sendTrackEvent } from '@edx/frontend-analytics';
-import SiteHeader from '@edx/frontend-component-site-header';
+import PropTypes from 'prop-types';
+import SiteHeader from '@edx/frontend-component-header';
```

Comment:
Just trying to understand these changes - is so much being removed because it is all contained in the siteheader?

Reply:
Yep, everything that's being removed here should be supplied by the up to date versions of [frontend-component-header](https://github.com/edx/frontend-component-header) and [frontend-component-footer](https://github.com/edx/frontend-component-footer)